homeassistant/components/xiaomi_miio/button.py

```diff
@@ -2,6 +2,9 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
+from typing import Any
```

Comment:
Remove the `Any` import now that `method_press_params: Consumable | None` does not need it

Reply:
Done